perf-tests/clusterloader2/pkg/framework/client/objects.go

```diff
@@ -186,27 +186,27 @@ func ListNodesWithOptions(c clientset.Interface, listOpts metav1.ListOptions) ([
 }
 
 // CreateNamespace creates a single namespace with given name.
-func CreateNamespace(c clientset.Interface, namespace string) error {
+func CreateNamespace(c clientset.Interface, namespace string, tenant string) error {
```

Comment:
Can we make the extra parameter "tenant string" to "tenants []string" and do a loop in the callee function? Then it would be able to duplicate workloads to multiple tenants within one test run.

Reply:
per offline sync, we are going to run multiple copies of perf-test for different tenants.

Reply:
Resolve this based on the discussion. We don't do it for now.